glue/flagging_script_glue/flagging_e894e5.py

```diff
@@ -1207,23 +1211,6 @@ def create_judicial_flag(df: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
-def create_match_flag(row: pd.Series) -> str:
-    """
-    Creates a column that says whether the buyer/seller id match.
-    Meant for apply().
-    Inputs:
-        row: from dataframe
-    Outputs:
-        value (str): whether the buyer and seller ID match
-    """
-    if row["buyer_id"] == row["seller_id"] and row["buyer_id"] != "Empty Name":
-        value = "Buyer ID and Seller ID match"
-    else:
-        value = "No match"
-
-    return value
-
-
```

Comment:
This was an unused function which was an exact duplicate of the `df_name_match` function.